### DISCUSSION TITLE
Pest Snapshots is it possible to ignore fields such as created_at?

### DISCUSSION THREAD

**Discussion by PaddingtonBrown** [pestphp/pest#1058] [Q&A]
Hi, when using Pest Snapshots for assert a database entry has been created correctly. Is it possible to skip fields such as created_at or updated_at which will not be the same every time?

e.g. something like
```
$post = Post::first();
expect($post)->toMatchSnapshot( <some how ignore created_at and updated_at>);
```

**Comment by MickProPay** [comment]
In the project we use when ever we test create events or update stuff we make use of the pest time freeze so that time is easier to check.
